src/test/scala/net/ruippeixotog/akka/testkit/specs2/AkkaMatchersSpec.scala

```diff
@@ -79,7 +79,7 @@ class AkkaMatchersSpec(implicit env: ExecutionEnv) extends TestKit(ActorSystem()
       probe.ref ! None
       (probe must matchTest) must beSuccessful
       probe.ref ! Some("")
-      (probe must matchTest) must beFailing("Received message 'Some\\(\\)' but undefined function for 'Some\\(\\)'")
+      (probe must matchTest) must beFailing("Received message 'Some\\(\\)' but undefined function for (Some\\(\\)|'Some\\(\\)')")
```

Comment:
I'm curious, does specs2 v4.5.1 print quotes only on Scala 2.13?

Reply:
Yes, but it is probably unintended. In Scala 2.13.0, `Option` extends `IterableOnce` and `TraversableOnce` is a (deprecated) alias to `IterableOnce`. This makes specs2 [stop adding quotes](https://github.com/etorreborre/specs2/blob/8305db76c5084e4b3ce5827ce23117f6fb6beee4/common/shared/src/main/scala/org/specs2/text/Quote.scala#L20) to `Option`s in failure messages, whereas before it would.